### ISSUE TITLE
Happymh 嗨皮漫画

### ISSUE THREAD

**Issue by kuronekonokuro** [tachiyomiorg/tachiyomi-extensions#11384]
### Source name

嗨皮漫画

### Source link

https://m.happymh.com/

### Language

Chinese

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a title with source name.
- [X] I have checked that the extension does not already exist on the [website extensions list](https://tachiyomi.org/extensions/) or the app.
- [X] I have checked that the extension does not already exist by searching the [GitHub repository](https://github.com/tachiyomiorg/tachiyomi-extensions/) and verified it does not appear in the code base.
- [X] I will fill out all of the requested information in this form.

**Comment by anenasa** [comment]
I am blocked by cloudflare and I am unable to find out how to bypass when I try to get image url :(
